app/Policies/PetPolicy.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Models\User;
+
+class PetPolicy
```

Comment:
Register this policy in AuthServiceProvider with something like this

`protected $policies = [ Pet::class => PetPolicy::class ];`